mlflow/utils/conda.py

```diff
@@ -10,6 +10,7 @@
 # Environment variable indicating a path to a conda installation. MLflow will default to running
 # "conda" if unset
 MLFLOW_CONDA_HOME = "MLFLOW_CONDA_HOME"
+MLFLOW_CONDA_CREATE_ENV_CMD = "MLFLOW_CONDA_CREATE_ENV_CMD"
```

Comment:
MLFLOW_CONDA_CREATE_ENV_CMD --> MLFLOW_CONDA_HOME_FOR_CREATE_ENV for naming consistency.
Also add some comments explaining when and how it's used with an example illustrated with mamba.

Reply:
I added the comment, but left the name as it was. I'm open to change that, but I wouldn't call it `*_HOME_*` because that refers to a path, while the `MLFLOW_CONDA_HOME_FOR_CREATE_ENV` is just a command.

For example, to use mamba for creating environments, you need to do:
> conda install mamba -n base -c conda-forge
> export MLFLOW_CONDA_HOME_FOR_CREATE_ENV=mamba
> mlflow run ...